### DISCUSSION TITLE
How can I get playlist data by its code?

### DISCUSSION THREAD

**Discussion by IvanPragma** [efroemling/ballistica#411] [Q&A]
I need to get the name of the playlist and the games it contains without using BombSquad. Should I send an HTTP request, what should it be?

**Comment by TheNotEasy** [comment]
The data of ONLY custom playlists is stored in the config.json file with the keys "Free-for-All Playlists", "Team Tournament Playlists". It is stored in "C:\\Users\\{user_name}\\AppData\\Local\\BallisticaCore\\config.json".

**Reply by IvanPragma** [reply]
> The data of ONLY custom playlists is stored in the config.json file with the keys "Free-for-All Playlists", "Team Tournament Playlists". It is stored in "C:\Users\{user_name}\AppData\Local\BallisticaCore\config.json".

Thanks for the answer, but it doesn't suit me. I need: having a playlist code generated by another player on another device, get the name and list of playlist games.
